### PR TITLE
fix(analytics): archived filter - BL21-612

### PR DESCRIPTION
**Pull request by johan-ds**

### CODE REVIEW

File: dart/.openapi-generator/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-6.5.0
\ No newline at end of file
+6.2.1
```
Comment (mrnvr) [inline dart/.openapi-generator/VERSION:1]: update your generator version

**Comment by johan-ds** [comment]
> `archived` parameter is missing from report endpoint. You can extract it from #112

On the '/organizations/{organization}/reports' ?

**Comment by mrnvr** [comment]
> > `archived` parameter is missing from report endpoint. You can extract it from #112
> 
> @mrnvr On the '/organizations/{organization}/reports' ?

yes

**Comment by johan-ds** [comment]
> > > `archived` parameter is missing from report endpoint. You can extract it from #112
> > 
> > 
> > @mrnvr On the '/organizations/{organization}/reports' ?
> 
> yes

I might get it wrong but it seems to me that 'archived' is already present in this endpoint

**Comment by mrnvr** [comment]
> > > > `archived` parameter is missing from report endpoint. You can extract it from #112
> > > 
> > > 
> > > @mrnvr On the '/organizations/{organization}/reports' ?
> > 
> > 
> > yes
> 
> I might get it wrong but it seems to me that 'archived' is already present in this endpoint

You're right. Sorry about that.

**Pull request merged**